templates/etc/postfix/main.cf.j2

```diff
@@ -81,9 +81,9 @@ smtp_sasl_security_options = {{ postfix_sasl_security_options }}
 smtp_sasl_mechanism_filter = {{ postfix_sasl_mechanism_filter }}
 {% endif %}
 {% if postfix_relaytls %}
-smtp_use_tls = yes
-smtp_tls_security_level = encrypt
-smtp_tls_note_starttls_offer = yes
+smtp_use_tls = {{ postfix_smtp_tls_security_level != 'none' | ternary('yes', 'no')}}
+smtp_tls_security_level = {{ postfix_smtp_tls_security_level }}
+smtp_tls_note_starttls_offer = {{ postfix_smtp_tls_note_starttls_offer | bool | ternary('yes', 'no') }}
```

Comment:
Note that this is only used when a `postfix_relayhost` is configured and if `postfix_relaytls`

@theel0ja
@agimenez

Reply:
If you want to use it without those options, you can do something like this:

```yml
postfix_raw_options:
  - |
    smtp_use_tls = yes
    smtp_tls_security_level = may
    smtp_tls_note_starttls_offer = yes
```

Reply:
Or do you guys think it should be outside the `if`'s

Reply:
I'd leave it inside the `if`, since for some reason someone may want not to use TLS, so flexibility is fine here IMO.

But I'd switch `smtp_use_tls` completely, as since 2.3, the recommended variable is `smtp_tls_security_level` instead (I am not sure anyone wants to use older versions in a critical piece of software like an SMTP server).

It depends on whether you want to potentially break old installations or not.